index.html

```diff
@@ -0,0 +1,125 @@
+          <span class="xIcon"
+            ><img src="resources/images/Union-2.svg" alt="x icon"
+          /></span>
+          <ul class="mobile">
+            <li><a href="#hello">Hello</a></li>
+            <li><a href="about.html"> About </a></li>
+            <li><a href="#events"> Events </a></li>
+            <li><a href="#speakers"> Speakers </a></li>
+            <li><a href="about.html#contact"> Contact </a></li>
+          </ul>
+        </div>
+      </nav>
+      <span id="hello" class="hello">"Hello! Beef-and-dairyans"</span>
+      <h1 class="summit-name">GREAT BRITISH CATTLE BAZAAR <br />2027</h1>
+      <p class="summit-description">
+        The world's largest livestock sale, with over 40,000 cattle sold annually, as well as 200 million litres of milk. And including a sparkling lineup of the biggest names in the production of beef animals and dairy herds.
+      </p>
+      <div class="date-location">
+        <h2 class="date">2027.10.12(THU) ~ 13(FRI)</h2>
+        <p class="location">
+          @ National Museum of Cattle, Beef Center and more
+        </p>
+      </div>
+    </section>
+    <section class="main-program" id="events">
+      <h2>Main Program</h2>
+      <ul class="events" id="events-dynamic"></ul>
+      <a href="#start" class="see-all desktop"
+        ><span>SEE THE WHOLE PROGRAM</span></a
+      >
+      <button type="button" class="join mobile">Join the Great British Cattle Bazaar 2027</button>
+    </section>
+    <section class="featuring" id="speakers">
+      <h2>Featured Speakers</h2>
+      <ul class="speakers" id="speakers-top"></ul>
+      <ul class="speakers less" id="speakers-dynamic"></ul>
+      <div id="more-container" class="mobile">
+        <button type="button" id="more-bttn" class="more-bttn">
+          <span id="more-txt">MORE</span
+          ><span
+            ><img
+              src="resources/images/arrowhead.svg"
+              alt="arrowhead"
+              width="30"
+              id="arrowhead"
+          /></span>
+        </button>
+      </div>
+    </section>
+      <section class="partners desktop">
+        <h2>Partners</h2>
+        <div class="partner-logos">
+          <img src="resources/images/behance.png" alt="icon"/>
+          <img src="resources/images/dropbox.png" alt="icon"/>
+          <img src="resources/images/soundcloud.png" alt="icon"/>
+          <img src="resources/images/ebay.png" alt="icon"/>
+          <img src="resources/images/wordpress.png" alt="icon"/>
+          <img src="resources/images/linkedin.png" class="mobile" alt="icon"/>
+          <img src="resources/images/instagram.png" class="mobile" alt="icon"/>
+        </div>
+      </section>
+    
+    <footer>
+      <div class="logo-footer">
+        <img src="resources/images/logo.png" alt="logo"/>
+      </div>
+      <div class="copyrigths">
+        © 2027 Creative British Cattle. <br />
+        Some Rights Reserved.
+      </div>
+    </footer>
```

Comment:
- [ ] Don't get me wrong I think you did a great job with this footer, but I would like you to try to make even better so please look at the examples and try to match more like the original like getting them closer to each other

Yours

![Screenshot from 2021-11-19 19-56-56](https://user-images.githubusercontent.com/62705947/142676859-531eb169-651b-4503-9c3b-983d63b18599.png)

Given design


![Screenshot from 2021-11-19 19-56-58](https://user-images.githubusercontent.com/62705947/142676876-7d3c31dc-9842-4f29-bfd3-49a73da8c95f.png)